### ISSUE TITLE
Removing some games causes them to permanently stay glitched on installed games

### ISSUE THREAD

**Issue by CrazyKingTrip** [gamejolt/issue-tracker#1115]

**Comment by Dramen-Lore** [comment]
I am not sure if the problem I am having is similar to yours, all that I know is that at first I had not problems with the Gamejolt Client but that was until recently, I had attempted to download 3 games at once on Gamejolt client, the 3 had repeatedly stopped downloading, I had to keep a close eye on them to ensure they downloaded only to find 2 games wouldn't download, so I decided to try and fix this by uninstalling the games, I only managed to uninstall 1 of them currently, I can not uninstall the other 2 games I sought to get, I've tried to uninstall them from the "Installed games" list but I can't no matter how much I try, I've attempted to uninstall them from their pages but the only option is to download them on windows. I've been trying to search for ways to fix this but to no avail, I assumed it was a bug and planned to make a report on it but I saw that it was best to not make a report on something that's already been reported and it seems my issue is similar to the one here however I'm not exactly sure. All I can say is I currently have 2 games in my installed games list that cannot be uninstalled at all.
